### PR TITLE
Providing clarity for more PRs, less commits

### PR DESCRIPTION
**Pull request by jaytiaki**

### CODE REVIEW

**Review by mrbobbytables — COMMENTED** [review]
Thanks, but this seems like unnecessarily adding more words to it? The original seems pretty clear.

Does anyone else have thoughts?

**Comment by hh** [comment]
@jaytiaki was confused by the initial wording. He's got fresh eyes from an early perspective here .Sometimes it takes more word to provide the clarity for folks who are just starting out.

File: contributors/guide/pull-requests.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ If you can extract whole ideas from your pull request and send those as pull req
 
 Kubernetes is a fast-moving codebase - lock in your changes ASAP with your small pull request, and make merges be someone else's problem.
 
-Multiple small pull requests are often better than multiple commits.
+Multiple small pull requests with fewer commits are often better than single pull requests with many commits.
```
Comment (hh) [inline contributors/guide/pull-requests.md:221]: The updated statement makes it clear that one type of pull request is better than another type of pull request. Keen if anyone has thoughts on another way to make this distinction! :)

**Comment by jaytiaki** [comment]
Thanks for your help @castrojo - who from this list should I ask for an approve from?
https://github.com/kubernetes/community/blob/master/contributors/guide/OWNERS#L8-L11

**Comment by jaytiaki** [comment]
Can I get your thoughts pls @guineveresaenger?
/assign @guineveresaenger

**Comment by castrojo** [comment]
/approve

It's been sitting here since April 4 so just approving this in as it's a minor change.

**Pull request closed without merging**